src/app/visual/datepicker/datepicker-visual.component.ts

```diff
@@ -4,20 +4,25 @@ import {
 } from '@angular/core';
 
 import {
+  AbstractControl,
   FormBuilder,
-  FormGroup,
   FormControl,
-  Validators,
-  AbstractControl
+  FormGroup,
+  Validators
 } from '@angular/forms';
 
 @Component({
   selector: 'datepicker-visual',
   templateUrl: './datepicker-visual.component.html'
 })
 export class DatepickerVisualComponent implements OnInit {
-  public selectedDate: Date = new Date('4/4/2017');
+  public disabled = false;
```

Comment:
Should we have a blank line before these public variables?

Reply:
I could go either way. We can come up with a style guide if you like.

Reply:
I personally like the line break and feel like it is more consistent with our other styles; however, I wouldn't consider it a blocker.